Assets/MXR.SDK/Runtime/Android/MXRAndroidSystem.cs

```diff
@@ -271,6 +271,27 @@ public class MXRAndroidSystem : IMXRSystem {
                 Debug.unityLogger.Log(LogType.Warning, TAG, "ConnectToWifiNetwork ignored. System is not available (not bound to messenger.");
         }
 
+        public void ConnectToEnterpriseWifiNetwork(EnterpriseWifiConnectionRequest enterpriseWifiConnectionRequest) {
+
+            if(enterpriseWifiConnectionRequest == null)
+                throw new ArgumentNullException(nameof(enterpriseWifiConnectionRequest));
+
+            if (messenger.IsBoundToService) {
+                if (LoggingEnabled)
+                    Debug.unityLogger.Log(LogType.Log, TAG, "ConnectToEnterpriseWifiNetwork called. Invoking over JNI: connectToEnterpriseWifiNetworkAsync");
+                    Debug.unityLogger.Log(LogType.Log, TAG, enterpriseWifiConnectionRequest.Ssid);
+                    Debug.unityLogger.Log(LogType.Log, TAG, enterpriseWifiConnectionRequest.Password);
```

Comment:
Do we want to leave all of these logs in? I think just logging that this was invoked is good enough for now.

At a minimum, we should certainly not make the default behavior to log passwords!

Reply:
Woops, forgot to remove them or add a TODO.

They are only temporary, but now that you mention it, do you think we should log the object attributes with the exception of the password?

Reply:
I think it's probably better to just remove the logging. If someone is debugging, they can always add it back in.

Reply:
Added a note in my TODOs to remove password logs post testing